services/catalog/src/simcore_service_catalog/cli.py

```diff
@@ -24,3 +27,52 @@ def run():
         "$ uvicorn simcore_service_catalog.main:the_app",
         fg=typer.colors.BLUE,
     )
+
+
+@main.command()
+def echo_dotenv(ctx: typer.Context, *, minimal: bool = True) -> None:
```

Comment:
maybe just `echo_env`? 🤔

Reply:
the files are commonly denoted as dot-env or `.env` :-) ... but it is a matter of taste.  In any case, the important thing is to use the same command across all the service settings